sfputil/main.py

```diff
@@ -1526,6 +1526,48 @@ def version():
     """Display version info"""
     click.echo("sfputil version {0}".format(VERSION))
 
+# 'target' subcommand
+@firmware.command()
+@click.argument('port_name', required=True, default=None)
+@click.argument('target', type=click.INT, required=True, default=None)
+def target(port_name, target):
+    """Select target end for firmware download 0-(local) \n
+                                               1-(remote-A) \n
+                                               2-(remote-B)
+    """
+    if target > 2:
```

Comment:
@AnoopKamath can you use click specific argument checking?

Reply:
Taken care.

```
root@sonic:/home/cisco# sfputil firmware target Ethernet0 3
Usage: sfputil firmware target [OPTIONS] PORT_NAME TARGET
Try "sfputil firmware target --help" for help.

Error: Invalid value for "TARGET": 3 is not in the valid range of 0 to 2.
```